src/Npgsql/PublicAPI.Shipped.txt

```diff
@@ -597,7 +597,6 @@ Npgsql.NpgsqlDataSource.OpenConnection() -> Npgsql.NpgsqlConnection!
 Npgsql.NpgsqlDataSource.OpenConnectionAsync(System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) -> System.Threading.Tasks.ValueTask<Npgsql.NpgsqlConnection!>
 Npgsql.NpgsqlDataSource.Password.set -> void
 Npgsql.NpgsqlDataSourceBuilder
-Npgsql.NpgsqlDataSourceBuilder.AddTypeResolverFactory(Npgsql.Internal.TypeHandling.TypeHandlerResolverFactory! resolverFactory) -> void
```

Comment:
Intentional? We shouldn't be modified Shipped (which represents 7.0), but rather adding *REMOVED* to Unshipped

Reply:
Oh, not sure what happened here. I'm pretty sure I did everything via `dotnet format`... Will revert.

Reply:
Fixed

Reply:
Still seeing various modifications in Shipped, any idea what these are about? In theory this file shouldn't be touched at all

Reply:
Yeah my bad, it's really fixed now...